src/layman/common/prime_db_schema/publications.py

```diff
@@ -230,7 +243,8 @@ def get_publication_infos_with_metainfo(workspace_name=None, pub_type=None, styl
 
     #########################################################
     # Put it together
-    sql_params = with_clause_params + select_params + where_params + order_by_params + pagination_params
+    with_publications_params = sum([calculated_column.params for calculated_column in calculated_columns], ())
```

Comment:
I prefer
`with_publications_params = tuple(param for column in calculated_columns for param in column.params)`
because of readability. Plus it's faster.
https://stackoverflow.com/a/51291027/5259610